src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/ActiveDirectoryAuthenticationProvider.cs

```diff
@@ -125,7 +209,9 @@ internal class ActiveDirectoryAuthenticationProvider : SqlAuthenticationProvider
             return new SqlAuthenticationToken(result.AccessToken, result.ExpiresOn);
         });
 
-        private async Task<AuthenticationResult> AcquireTokenInteractive(IPublicClientApplication app, string[] scopes, Guid connectionId, string userId)
+
+        private async Task<AuthenticationResult> AcquireTokenInteractiveDeviceFlow(IPublicClientApplication app, string[] scopes, Guid connectionId, string userId,
```

Comment:
Do we use the `Async `suffix for private methods?

Reply:
I'll add it for clarity!